emoji/src/test/java/com/vanniktech/emoji/EmojiManagerTest.java

```diff
@@ -68,4 +82,10 @@
 
     assertThat(EmojiManager.getInstance().findEmoji("")).isNull();
   }
+
+  @Test public void findAllEmojisEmpty() {
```

Comment:
This is already [entirely covered] by the tests in the `EmojiHandlerTest`. Should i test it nevertheless here?